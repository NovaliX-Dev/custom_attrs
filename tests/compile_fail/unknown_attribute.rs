use custom_attrs::CustomAttrs;

#[derive(CustomAttrs)]
#[attr(pub a: usize)]
enum Enum {
    #[attr(a = 3)]
    #[attr(b = 3)]
    Variant1,

    #[attr(a = 3)]
    Variant2
}

fn main() {
    // let _a = Enum::Variant1.get_a();
}