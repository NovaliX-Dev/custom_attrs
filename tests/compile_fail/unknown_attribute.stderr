error: Unknown attribute.
 --> tests/compile_fail/unknown_attribute.rs:7:12
  |
7 |     #[attr(b = 3)]
  |            ^
